mne/decoding/search_light.py

```diff
@@ -65,6 +65,8 @@ def fit(self, X, y):
             X.shape = (n_samples, n_features_1, n_features_2, n_tasks)
         y : array, shape (n_samples,) | (n_samples, n_targets)
             The target values.
+        **fit_params : dict of string -> object
```

Comment:
just `dict` is fine here, the fact that they get passed is clear in the description